README.md

```diff
@@ -138,6 +224,8 @@ This Integration Guide should:
 
 ## Authentication
 
+> One might also link or integrate this guide: [https://dome-marketplace.github.io/iam-guide/](https://dome-marketplace.github.io/iam-guide/)
```

Comment:
The iam-guide is indeed very useful reading! Thanks!

Either the iam-guide or this guide should offer a step by step tutorial describing the usage of the three components (Certificate Issuer aaS, Certificate Signer & Wallet) that will be provided by DOME as the iam-guide specifies.

Reply:
The instructions of the iam-guide will be either integrated or linked in the authentication section of this guide. But this PR is just about the introduction. I just already added that link for the author of the section.